core/__tests__/bin/generate.ts

```diff
@@ -51,7 +51,7 @@ describe("bin/generate", () => {
   test("the generate command can filter the list of templates", async () => {
     const command = new Generate();
     await command.run({
-      params: { template: "group", list: true },
+      params: { template: "group", list: "true" },
     });
```

Comment:
The args to CLI commands are really tricky... they can be strings or booleans... typing them is quite odd.  I went with keeping them all strings